### ISSUE TITLE
Improve error message when -i and --amend --no-edit are passed together

### ISSUE THREAD

**Issue by petroseskinder** [twosigma/git-meta#711]
In vanilla git, one cannot pass `-i` and `--amend --no-edit`. The error response with vanilla git is 

```
fatal: Only one of --include/--only/--all/--interactive/--patch can be used.
```
With `git-meta` the error message is much less clear
```
git meta commit -ai --amend --no-edit
TypeError: editMessage is not a function
    at Object.<anonymous> (/.../lib/util/commit.js:2154:32)
    at Generator.next (<anonymous>)
    at onFulfilled (/.../node_modules/co/index.js:65:19)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

As expected, when the `-i` or `-no-edit` is not passed `git-meta` works as expected. We should improve git meta's error message to reflect git's error messages.

**Comment by petroseskinder** [comment]
I think we just need to add more requirements in the `commit` argument parsing
https://github.com/twosigma/git-meta/blob/abce63fc037827653a0c2a11d70ea51ecc6731ec/node/lib/cmd/commit.js#L71

**Comment by novalis** [comment]
Fixed in #794
